backend/src/zango/apps/accesslogs/apps.py

```diff
@@ -2,8 +2,7 @@
 
 
 class AccesslogsAppConfig(AppConfig):
-
     name = "zango.apps.accesslogs"
 
     def ready(self):
-        import zango.apps.accesslogs.signals
```

Comment:
@DevilsAutumn 
importing signals will be required

please check it in other places also

Reply:
okay